clients/java/src/main/java/io/camunda/client/impl/command/UnassignUserFromGroupCommandImpl.java

```diff
@@ -27,21 +27,21 @@
 
 public class UnassignUserFromGroupCommandImpl implements UnassignUserFromGroupCommandStep1 {
 
-  private final long groupKey;
+  private final String groupId;
   private final HttpClient httpClient;
   private final RequestConfig.Builder httpRequestConfig;
-  private long userKey;
+  private String username;
 
-  public UnassignUserFromGroupCommandImpl(final long groupKey, final HttpClient httpClient) {
-    this.groupKey = groupKey;
+  public UnassignUserFromGroupCommandImpl(final String groupId, final HttpClient httpClient) {
+    this.groupId = groupId;
     this.httpClient = httpClient;
     httpRequestConfig = httpClient.newRequestConfig();
   }
 
   @Override
-  public UnassignUserFromGroupCommandStep1 userKey(final long userKey) {
-    ArgumentUtil.ensureNotNull("userKey", userKey);
-    this.userKey = userKey;
+  public UnassignUserFromGroupCommandStep1 username(final String username) {
+    ArgumentUtil.ensureNotNull("username", username);
```

Comment:
🙃 We could either remove this check here, or check for not null nor empty and remove it from the send method.